include/nuttx/fs/fs.h

```diff
@@ -224,6 +224,20 @@ struct geometry
   blksize_t geo_sectorsize;   /* Size of one sector */
 };
 
+struct partition_info_s
+{
+  uint32_t        magic;        /* File system magic, 0 for RAW
+                                 * (see <sys/statfs.h>) */
+  size_t          length;       /* Partition size in bytes */
```

Comment:
redundanct info? can computed from numsectors and sectorsize.

Reply:
Done.